packages/amplify-migration-tests/tsconfig.tests.json

```diff
@@ -3,6 +3,7 @@
     "target": "es6",
     "module": "commonjs",
     "sourceMap": true,
+    "esModuleInterop": true,
```

Comment:
why this one?

Reply:
Compiler said I need that if I want `import strip from 'strip-ansi';` . 
This flag is pretty common https://github.com/aws-amplify/amplify-cli/search?q=%22esModuleInterop%22%3A+true%2C .